src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/attn_memcpy.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+#include <openvino/core/type/element_type.hpp>
+#include "utils/plain_tensor.hpp"
+
+namespace InferenceEngine {
+namespace Extensions {
+namespace Cpu {
+namespace XARCH {
+
+void attn_memcpy(const ov::intel_cpu::PlainTensor& k_input,
```

Comment:
should add to cross_compile:
https://github.com/openvinotoolkit/openvino/blob/bdfe1cb291a51cb6de2aa7796d060344880e4286/src/plugins/intel_cpu/CMakeLists.txt#L162-L167